core/client/fs/src/test/java/alluxio/client/block/policy/LocalFirstPolicyTest.java

```diff
@@ -120,6 +131,33 @@ public void chooseClosestTier() throws Exception {
     assertEquals("node4", chosen.getTieredIdentity().getTier(0).getValue());
   }
 
+  @Test
+  public void chooseLocalAccessibleDomainSocket() throws Exception {
```

Comment:
Does this test belong in `LocalFirstPolicyTest`?

Reply:
it doesn't matter much as `LocalFirstPolicy::nearest` is a thin wrapper over `BlockLocationUtils::nearest`. i'll move it to a to a new class for `BlockLocationUtilsTest` to be more granular

Reply:
Thanks, I was more concerned about putting the test in a reasonable place and not the test correctness. It seemed weird to test for this in a policy since this should work regardless of which policy is used.

Reply:
done